src/app/tables/table.component.html

```diff
@@ -83,6 +85,11 @@ <h2 class="table-title" *ngIf="title">
                   </ng-template>
                 </td>
               </ng-container>
+              <td *ngIf="nodeStoryUrls[item[uidColumn]] !== undefined">
+                <a [href]="nodeStoryUrls[item[uidColumn]]" target="_blank">
```

Comment:
Second part of the suggestion:
```suggestion
                <ng-container *ngIf="nodeStoryUrls[item[uidColumn]]""
                  <a [href]="nodeStoryUrls && nodeStoryUrls[item[uidColumn]]" target="_blank">
                </ng-container>
```